index.html

```diff
@@ -66,35 +66,35 @@ <h2 class="lead secondary-heading">
         <p>Julia was designed from the beginning for <a href="/benchmarks/">high performance</a>.
         Julia programs compile to efficient native code for <a href="/downloads/#support_tiers">multiple platforms</a> via LLVM.</p>
       </div>
       <!-- 2 -->
       <div class="col-lg-4 col-md-6 feature">
         <h3>Dynamic</h3>
-        <p>Julia is <a href="https://docs.julialang.org/en/v1/manual/types/">dynamically typed</a>, feels like a scripting language, and has good support for <a href="https://docs.julialang.org/en/v1/stdlib/REPL/">interactive</a> use.</p>
+        <p>Julia is <a href="https://docs.julialang.org/en/v1/manual/types/">dynamically typed</a>, feels like a scripting language, and has good support for <a href="https://docs.julialang.org/en/v1/stdlib/REPL/">interactive</a> use. Julia has a <a href="https://docs.julialang.org/en/v1/manual/types/">rich language of descriptive datatypes</a>, and type declarations can be used to clarify and solidify programs.</p>
+</p>
       </div>
       <!-- 3 -->
       <div class="col-lg-4 col-md-6 feature">
-        <h3>Optionally typed</h3>
-        <p>Julia has a <a href="https://docs.julialang.org/en/v1/manual/types/">rich language of descriptive datatypes</a>, and type declarations can be used to clarify and solidify programs.</p>
+        <h3>Reproducible</h3>
+        <p><a href="https://julialang.github.io/Pkg.jl/v1/environments/">Immutable environments</a> let you recreate the same Julia environment every time, across platforms. When packages have binary dependencies, we <a href="https://julialang.org/blog/2019/11/artifacts/">build them for every platform</a> so it installs in seconds and is always linked correctly.</p>
       </div>
       <!-- 4 -->
       <div class="col-lg-4 col-md-6 feature">
-        <h3>General</h3>
-        <p>Julia uses <a href="https://docs.julialang.org/en/v1/manual/methods/">multiple dispatch</a> as a paradigm, making it easy to express many object-oriented and <a href="https://docs.julialang.org/en/v1/manual/metaprogramming/">functional</a> programming patterns.
-        It provides <a href="https://docs.julialang.org/en/v1/manual/networking-and-streams/">asynchronous I/O</a>, <a href="https://github.com/JuliaDebug/Debugger.jl">debugging</a>, <a href="https://docs.julialang.org/en/v1/stdlib/Logging/">logging</a>, <a href="https://docs.julialang.org/en/v1/manual/profile/">profiling</a>, a <a href="https://docs.julialang.org/en/v1/stdlib/Pkg/index.html">package manager</a>, and more.</p>
+        <h3>Multiple Dispatch</h3>
+        <p>Julia uses <a href="https://docs.julialang.org/en/v1/manual/methods/">multiple dispatch</a> as a paradigm, making it easy to express many object-oriented and <a href="https://docs.julialang.org/en/v1/manual/metaprogramming/">functional</a> programming patterns. The talk on the <a href="https://www.youtube.com/watch?v=kc9HwsxE1OY">Unreasonable Effectiveness of Multiple Dispatch</a> explains why it works so well.
```

Comment:
This should use the "functions" chapter as the link for "functional" instead of metaprogramming. Macros are not really related to functional programming.

Reply:
To make the section name an adjective like the others, could rename it "Flexible"?

Reply:
Or "Composable"?